source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,79 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.rgh-edit-comment, [aria-label="Edit comment"]'],
+	['d', '[aria-label="Delete comment"]'],
+	['h', '[aria-label="Hide comment"]'],
+	['j', ''],
+	['k', '']
+]);
+
+function commentHash(comment: HTMLElement): string {
+	// What do I call this var??
+	const reviewComment = comment.closest('.js-comment[id^="pullrequestreview-"]');
+	const {hash} = reviewComment ? reviewComment.querySelector<HTMLAnchorElement>('a.js-timestamp')! : comment.querySelector<HTMLAnchorElement>('a.js-timestamp')!;
+
+	return reviewComment ? hash + '-body-html' : hash;
+}
+
+function runShortcuts(event: delegate.Event<KeyboardEvent>): void {
+	if (!shortcutClass.has(event.key) || event.target instanceof HTMLTextAreaElement || event.target instanceof HTMLInputElement) {
+		return;
+	}
```

Comment:
2/4

```suggestion
	if (isEditable(event.target)) {
		return;
	}
```

Reply:
Unrelated I took the code from here.
https://github.com/sindresorhus/refined-github/blob/0fa514dbeeab6395ef095cff80beac41443a6574/source/features/improve-shortcut-help.tsx#L50

We should do the same there?

Reply:
Sure